functors_task_1/student/include/impl.h

```diff
@@ -8,6 +8,18 @@
  */
 struct BeerOrganizer
 {
+    BeerBrand operator()()
+    {
+        ++brandIndex;
+        if (brandIndex == static_cast<size_t>(BeerBrand::Max))
```

Comment:
please specify the possible range explicitly with "from" and "to" constexpr variables

Reply:
done